.github/workflows/java-kotlin-aot.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     timeout-minutes: 15
     strategy:
       matrix:
-        java-version: ['17', 'dev']
```

Comment:
@LesiaChaban , is this the older version of GitHub Actions  for GraalVM? 
I know to upgrade, it should be replaced with:
```
  java-version: '21.0.1'
  distribution: 'graalvm'
  github-token: ${{ secrets.GITHUB_TOKEN }}
  cache: 'maven'
```

Reply:
Please see https://github.com/marketplace/actions/github-action-for-graalvm#migrating-from-graalvm-223-or-earlier-to-the-new-graalvm-for-jdk-17-and-later

Reply:
Hm...I think I made the changes in the right way